contracts/controller/Controller.sol

```diff
@@ -257,6 +257,20 @@ contract Controller is ControllerInterface {
         return schemes[_scheme].permissions;
     }
 
+    function getGlobalConstraintParameters(address _globalConstraint,address) external view returns(bytes32) {
```

Comment:
Please return here 2 values  : pre and post params  .
As gc can have both.

Reply:
As I read the code, both pre and post will have exactly the same params.  Is that not the case?